Documentation/Tools/HolographicRemoting.md

```diff
@@ -57,6 +57,20 @@ You can also temporarily remove the adapter to workaround your issue via the fol
 1. Search for DotNetWinRT.dll in your assets list in Unity and either delete the DLL or delete the Plugins (MRTK 2.2 or earlier) or Dependencies (MRTK 2.3 or later) folder that contains it a few levels up. That should remove these conflicting namespaces, while keeping MRTK around
 1. If you run the MRTK Configurator again, make sure you don't re-enable MSBuild for Unity
 
+### Failure to find dotnet.exe
```

Comment:
Should this go alongside the first list, as a fourth source of potential issues?

Reply:
Yes, thanks for calling this out! I intended to put it in the list but misplaced it.